Dockerfile

```diff
@@ -1,12 +1,7 @@
-FROM gcr.io/google.com/cloudsdktool/cloud-sdk:slim
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
 
-RUN apt-get update && \
-    apt-get install -y curl gnupg && \
-    echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
-    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg  add - && \
-    apt-get update -y && \
-    apt-get install google-cloud-sdk-gke-gcloud-auth-plugin -y && \
-    apt-get -y install kubectl
+RUN apk --update add curl && \
+    apk add kubectl
```

Comment:
Really minor - it might be worth adding the `--no-cache` flag when adding packages. Might not make a difference to size but worth removing the cache after it's no longer needed? Similar to what we do in the tiny proxy Dockerfile https://github.com/ONSdigital/eq-runner-docker-tinyproxy/blob/main/Dockerfile#L7

Reply:
Good point. Instead of add `--no-cache` to every package, we can run `rm /var/cache/apk/*` after installing all packages. It would be equivalent to add `--no-cache` to every add command.

Reply:
Added this in to Dockerfile, thanks 👍